Source/ImageGlass/LocalSetting.cs

```diff
@@ -87,6 +87,12 @@ public static class LocalSetting
         /// </summary>
         public static string InitialInputImageFilename { get; set; } = "";
 
+        /// <summary>
+        /// The current image sorting order. This is a temporary override which takes place when respecting
+        /// the user's sorting order from Windows Explorer.
+        /// </summary>
+        public static ImageOrderBy ImageLoadingOrder { get; set; }
```

Comment:
Because it's a temporary override, let's name it as `TempImageLoadingOrder` or `ExplorerImageOrder` 🤔